components/Feedback/CommentTeaser.js

```diff
@@ -0,0 +1,227 @@
+import React, { Fragment } from 'react'
```

Comment:
on second thought profile might be quite different because the author is given by the context. however it would be nice to build something generic that allows to e.g. disable author, tag or article context base on the wide context the teaser is in and document how to use it in the [styleguide](https://styleguide.republik.ch/components/commentteaser)